### DISCUSSION TITLE
ERROR: Cannot find module '@twind' or its corresponding type declarations.

### DISCUSSION THREAD

**Discussion by NicWille** [freshframework/fresh#419] [Q&A] [answered]
Im using vscode. i have alot of red squigley lines , with anything related to the imports "preact" or "twind" . i tried configuring deno in the vsvode/settings.json, and tried many other things.  I have search 3 other issues in deno but nothing seems to help.

**Comment by lucacasonato** [comment]
Did you try restarting VS Code? You might also just need to start the project for the first time so that all of the dependencies are downloaded.

Can you also screenshot the error?

**Comment by nikableh** [comment]
I've had a similar problem, and I solved it by disabling the Kite extension for VSCode.

It is useless anyway if you use TypeScript. There must be some way to selectively turn off extensions for individual projects.

**Reply by zicklag** [reply]
In VSCode you can choose to "Disable (Workspace)" instead of just "Disable" to disable it only for a specific project.

**Comment by NicWille** [comment, marked as answer]
solved it by updating my version of vs-code, and typescript. but it was my older version of vs-code that was the main problem
